modules/openy_features/openy_prgf/modules/openy_prgf_event_listing/config/install/views.view.listing_event_posts.yml

```diff
@@ -12,14 +13,15 @@ dependencies:
     - user
     - verf
     - views_infinite_scroll
+_core:
+  default_config_hash: aWBfhdf0ZQ9f2R4OiQLWMmn8O_VSkLH8LCQqB-nj8uc
 id: listing_event_posts
 label: 'Listing Event Posts'
 module: views
 description: 'Latest news posts list for landing page.'
 tag: ''
 base_table: node_field_data
 base_field: nid
-core_version_requirement: ^8 || ^9
```

Comment:
Should this be removed @podarok?

Reply:
yes, all good here. It was a fail add during autoconfig fix phase for D8->D9 upgrade